packages/suite/src/components/guide/GuideDefault.tsx

```diff
@@ -40,7 +42,7 @@ const FeedbackButton = styled.button`
 
     :hover,
     :focus {
-        background: ${({ theme }) => darken(theme.HOVER_DARKEN_FILTER, theme.BG_WHITE)};
+        background: ${({ theme }) => theme.backgroundTertiaryPressedOnElevation0};
```

Comment:
nit: use `backgroundTertiaryPressedOnElevation1` like in the `SupportFeedbackSection`